ui/dist/index.html

```diff
@@ -0,0 +1,956 @@
+            }
+            .prompt-modifier-tag:hover {
+                background: black;
+                color: white;
+            }
+            #editor-modifiers-entries .prompt-modifier-tag {
+                background: #110f0f;
+                color: rgb(212, 212, 212);
+                margin-bottom: 4pt;
+                font-size: 10pt;
+            }
+            #editor-modifiers-entries .prompt-modifier-tag:hover {
+                background: rgb(163, 163, 163);
+                color: black;
+            }
+            #editor-modifiers .editor-modifiers-leaf {
+                padding-top: 10pt;
+                padding-bottom: 10pt;
+            }
+            #preview {
+                margin-left: 20pt;
+            }
+            img {
+                box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.15), 0 6px 20px 0 rgba(0, 0, 0, 0.15);
+            }
+            .line-separator {
+                background: rgb(56, 56, 56);
+                height: 1pt;
+                margin: 15pt 0;
+            }
+            #editor-inputs-tags-container {
+                margin-top: 5pt;
+                display: none;
+            }
+            #server-status {
+                float: right;
+            }
+            #server-status-color {
+                width: 8pt;
+                height: 8pt;
+                border-radius: 4pt;
+                background-color: rgb(128, 87, 0);
+                /* background-color: rgb(197, 1, 1); */
+                float: left;
+                transform: translateY(15%);
+            }
+            #server-status-msg {
+                color: rgb(128, 87, 0);
+                padding-left: 2pt;
+                font-size: 10pt;
+            }
+            #preview-prompt {
+                font-size: 16pt;
+                margin-bottom: 10pt;
+            }
+        </style>
+
+        <!-- All of the original DOM -->
+        <div id="container">
+            <div class="flex-container">
+                <div id="editor" class="col-50">
+                    <div id="meta">
+                        <div id="server-status">
+                            <div id="server-status-color">&nbsp;</div>
+                            <span id="server-status-msg">Stable Diffusion is starting..</span>
+                        </div>
+                        <h1>Stable Diffusion UI <small>v2.01 (beta)</small></h1>
+                    </div>
+                    <div id="editor-inputs">
+                        <div id="editor-inputs-prompt" class="row">
+                            <label for="prompt">Prompt</label>
+                            <textarea id="prompt" class="col-free">a photograph of an astronaut riding a horse</textarea>
+                        </div>
+
+                        <div id="editor-inputs-init-image" class="row">
+                            <label for="init_image"><b>Initial Image:</b> (optional) </label> <input id="init_image" name="init_image" type="file" />
```

Comment:
in the original code there was an extraneous `</button>`  the build complained so i removed it.

Reply:
that is the only change I made to the original html file, other than moving everything into the body.